[2024-05-11T10:13:51.234+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-11T10:13:51.402+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v4.task3 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-11T10:13:51.435+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v4.task3 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-11T10:13:51.437+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-11T10:13:51.575+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task3> on 2024-05-08 00:00:00+00:00
[2024-05-11T10:13:51.618+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v4', 'task3', 'scheduled__2024-05-08T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpn44jowno']
[2024-05-11T10:13:51.640+0000] {standard_task_runner.py:91} INFO - Job 31: Subtask task3
[2024-05-11T10:13:51.758+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=8421) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-11T10:13:51.791+0000] {standard_task_runner.py:63} INFO - Started process 8426 to run task
[2024-05-11T10:13:53.407+0000] {task_command.py:426} INFO - Running <TaskInstance: my_first_dag_v4.task3 scheduled__2024-05-08T00:00:00+00:00 [running]> on host 48bfb4afd9cd
[2024-05-11T10:13:54.630+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='edo' AIRFLOW_CTX_DAG_ID='my_first_dag_v4' AIRFLOW_CTX_TASK_ID='task3' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-08T00:00:00+00:00'
[2024-05-11T10:13:54.642+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-11T10:13:54.762+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-11T10:13:54.765+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo third one matey']
[2024-05-11T10:13:54.847+0000] {subprocess.py:86} INFO - Output:
[2024-05-11T10:13:54.855+0000] {subprocess.py:93} INFO - third one matey
[2024-05-11T10:13:54.859+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-11T10:13:54.863+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-11T10:13:55.029+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v4, task_id=task3, run_id=scheduled__2024-05-08T00:00:00+00:00, execution_date=20240508T000000, start_date=20240511T101351, end_date=20240511T101355
[2024-05-11T10:13:55.266+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-11T10:13:55.650+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
